packages/playground/src/components/dynamic_tabs.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-tabs v-show="!hideTabs" v-model="activeTab" align-tabs="center" color="secondary" class="mb-6">
+  <v-tabs v-if="!hideTabs" v-model="activeTab" align-tabs="center" color="secondary" class="mb-6">
```

Comment:
i don't think this should prevent the tabs from appearing, and this change may affect other components that are using the dynamic tabs component 

i think we should pass here true to target only the deploymment list and to avoid unwanted changes 
https://github.com/threefoldtech/tfgrid-sdk-ts/blob/3b9d93cce5eb5c028034ae517811986b023b09dd/packages/playground/src/weblets/tf_deployment_list.vue#L12